bedrock/firefox/firefox_details.py

```diff
@@ -327,12 +328,12 @@ class FirefoxAndroid(_ProductDetails):
     }
 
     store_url = settings.GOOGLE_PLAY_FIREFOX_LINK
-    aurora_url_base = ('https://archive.mozilla.org/pub/mobile/nightly/'
-                       'latest-mozilla-aurora-android')
-    aurora_urls = {
-        'api-9': aurora_url_base + '-api-9/fennec-%s.multi.android-arm.apk',
-        'api-15': aurora_url_base + '-api-15/fennec-%s.multi.android-arm.apk',
-        'x86': aurora_url_base + '-x86/fennec-%s.multi.android-i386.apk',
+    archive_url_base = ('https://archive.mozilla.org/pub/mobile/nightly/'
+                        'latest-mozilla-%s-android')
+    archive_urls = {
+        'api-9': archive_url_base + '-api-9/fennec-%s.multi.android-arm.apk',
```

Comment:
api-9 is Android 2.3.x and I don't think we still support it, looking at our FTP:
https://archive.mozilla.org/pub/mobile/releases/47.0/
https://archive.mozilla.org/pub/mobile/releases/48.0/

It seems we only support api-15 now (Android 4.4+)